geographies/serializers.py

```diff
@@ -0,0 +1,49 @@
+from .models import * 
```

Comment:
use explicit relative imports only if it's too complex.

Reply:
Explicit imports added